packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -356,7 +357,7 @@ async function applyCategoryTemplate(
         } else {
           increment = limit;
         }
-        if (to_budget + increment < budgetAvailable || !priority) {
+        if (increment < budgetAvailable || !priority) {
```

Comment:
Was this change intentional?

Reply:
I think this was in the PR that was just merged.  I made the change to test it.  I can revert it back in this PR.